### ISSUE TITLE
[ERR][get-sdk] Failed to get Windows SDK!

### ISSUE THREAD

**Issue by anacondaq** [reksar/vsget#2]
```
D:\Desktop\vsget>vsget.bat my_portable_compiler

Getting 7-zip archiver
- Ensuring destination path exists ... OK
- Ensuring destination path empty ... OK
- Getting 7-zip self-extracting archive ... OK
- Getting util archiver ... OK
- Extracting 7-zip with util ... OK

Getting vc-x64-x64.14.33.31629
- Getting Microsoft.VC.14.33.17.3.CRT.Headers.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.32.17.2.CRT.Redist.X64.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.33.17.3.CRT.Source.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.33.17.3.CRT.x64.Desktop.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.33.17.3.CRT.x64.Store.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.33.17.3.Tools.HostX64.TargetX64.base
-   Downloading ... OK
-   Extracting ... OK
- Getting Microsoft.VC.14.33.17.3.Tools.HostX64.TargetX64.Res.base.enu
-   Downloading ... OK
-   Extracting ... OK

Getting msbuild-x64.17.3.1.2241501
- Getting Microsoft.Build
-   Downloading ... OK
-   Extracting ... OK
- Getting b98a3e4
-   Downloading ... OK
-   Extracting ... OK
- Getting fbb74fe
-   Downloading ... OK
-   Extracting ... OK
- Getting e17b4f9
-   Downloading ... OK
-   Extracting ... OK
- Getting c03dd5e
-   Downloading ... OK
-   Extracting ... OK

Getting Windows SDK
- Downloading Windows SDK ISO ... OK
- Extracting SDK Installers ... OK
- Extracting SDK Components

- Collecting SDK components
[ERR][get-sdk] Failed to get Windows SDK!
```

**steps to reproduce:**
1. git clone latest repo (0.1.4)
2. run cmd as admin
3. go to folder (cd vsget D:\vsget)
4. vsget.bat my_portable_compiler
5. -> error happened.

**Comment by reksar** [comment]
The main problem was extracting the SDK components (solved in c928cef07925a5833bd81e78cbee7cc2b42bb9b1).

@anacondaq I hope everything is OK now)

**Comment by anacondaq** [comment]
yes, everything is okay =) thx
